etc/frontend.json

```diff
@@ -1,6 +1,6 @@
 {
     "backend-api": {
-        "url": "http://127.0.0.1:8383"
+        "url": "http://minion-backend:8383"
```

Comment:
This is changed as a result of the official recommendation being to run minion front and backends separately.